addons/zeus/CfgEventHandlers.hpp

```diff
@@ -4,3 +4,11 @@ class Extended_PreInit_EventHandlers {
         init = QUOTE(call COMPILE_FILE(XEH_preInit));
     };
 };
+
+class Extended_InitPost_EventHandlers {
+    class AllVehicles {
+        class ADDON {
+            init = QUOTE(call FUNC(addObjectToCurator));
```

Comment:
Aha, of course, I assumed params wouldn't like that.

Also forgot this is `string`, not `code`. Looks good :+1: